src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -950,22 +942,26 @@ async def start_grpc_server(
     return server
 
 
-async def run_grpc_server(
-    engine: AsyncLLMEngine,
-    args: argparse.Namespace,
-    *,
-    disable_log_stats: bool,
-) -> None:
-    assert args is not None
+async def run_grpc_server(args: argparse.Namespace) -> None:
+    from vllm.entrypoints.openai.api_server import AsyncEngineRPCClient
 
-    server = await start_grpc_server(engine, args)
+    rpc_server_port = vllm.envs.VLLM_RPC_PORT
+    logger.info("gRPC server connecting to async engine on port %s", rpc_server_port)
+    async_engine_client = AsyncEngineRPCClient(rpc_server_port)
```

Comment:
I think we need to share the `AsyncEngineClient` from the http server because it might not be an instance of `AsyncEngineRPCClient` .. if running in local mode

Reply:
Done